src/runtime_src/core/pcie/driver/linux/xocl/devices.h

```diff
@@ -2951,8 +2945,8 @@ struct xocl_subdev_map {
 		.flags = XOCL_DSAFLAG_DYNAMIC_IP |			\
 			XOCL_DSAFLAG_VERSAL_ES3 |			\
 			XOCL_DSAFLAG_VERSAL,				\
-		.subdev_info = RES_USER_VERSAL_VSEC_ES3,		\
-		.subdev_num = ARRAY_SIZE(RES_USER_VERSAL_VSEC_ES3),	\
+		.subdev_info = RES_USER_VERSAL_VSEC,		\
```

Comment:
I don't understand this changes. The background of VSEC_ES3 was due to a dma bug in xdma that only 2 channels are working for es3 legacy shell. If we switch to regular versal vsec, which will enable 4 channels. Have you tested the dmatest on most recent es3 shell (gen3x16_xdma)?